ethereum/contracts/BeefyLightClient.sol

```diff
@@ -140,10 +140,9 @@ contract BeefyLightClient {
 
     // We must ensure at least one block is processed every session,
     // so these constants are checked to enforce a maximum gap between commitments.
-    uint64 public constant NUMBER_OF_BLOCKS_PER_SESSION = 2400;
-    uint64 public constant ERROR_AND_SAFETY_BUFFER = 10;
-    uint64 public constant MAXIMUM_BLOCK_GAP =
-        NUMBER_OF_BLOCKS_PER_SESSION - ERROR_AND_SAFETY_BUFFER;
+    // TODO: verify SAFETY_BUFFER
```

Comment:
don't forget to create issue flagged as security with this if you don't fix it before merge

Reply:
ah cool yeh if the increment-every-session is now guaranteed by beefy, then this sounds like a safe assumption.